backend/printcmm.ml

```diff
@@ -164,7 +164,7 @@ let operation d = function
   | Caddi -> "+"
   | Csubi -> "-"
   | Cmuli -> "*"
-  | Cmulhi -> "*h"
+  | Cmulhi { signed } -> "*h"^(if signed then "s" else "")
```

Comment:
In `Cfg`, `u` is added for unsigned while `s` is added here fore signed;
shouldn't we just always have a suffix to enforce uniformity?

Reply:
Ah, thanks, this one was a mistake. I was trying to match upstream output for the signed version.